Simplenote/Information/NoteInformationController.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
+import SimplenoteFoundation
+
+final class NoteInformationController {
+
+    /// Row
+    ///
+    enum Row {
+        /// Metric row
+        ///
+        case metric(title: String, value: String?)
+    }
+
+    /// Observer sends changes in rows
+    /// When assigned, it sends current state
+    ///
+    var observer: (([Row]) -> Void)? {
+        didSet {
+            observer?(allRows())
+
+            if observer == nil {
+                stopListeningForChanges()
+            } else {
+                startListeningForChanges()
+            }
```

Comment:
If the observer was non nil, and gets replaced, this one may lead to a a dual call to `startListenintForChanges`.

How abooout initializing the "listen for changes" during init, and maybe not tie that up to the observer? WDYT?

Reply:
My idea was to have `EntityObserver` observing things only when there is someone listening to updates on `NoteInformationController`.
I can set / remove delegate for `EntityObserver` instead of recreating it 👍